test/evergreen.yml

```diff
@@ -2528,6 +2507,51 @@ tasks:
           # run for 10 minutes.
           format_test_script_args: -t 10 rows=10000 ops=50000
 
+  - name: format-smoke-test
+    exec_timeout_secs: 3600
```

Comment:
Is this still needed? It's probably worth adding a comment to explain why, if so.

Reply:
Removed this line in 2cb26c5